application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1900,6 +1900,8 @@ type RealTimeEstimate {
 type RentalVehicle implements Node & PlaceInterface {
   "If true, vehicle is currently available for renting."
   allowPickupNow: Boolean
+  "The date and time when any rental of the vehicle must be completed."
```

Comment:
This wording is from the GBFS specification, but for me it's a bit vague if this means that the vehicle should be rented or returned before this time. The spec elaborates it in more words. Perhaps we could just say "The vehicle should be returned before this deadline."?

Reply:
yes, I replaced the description with your suggestion.